docs/guides/configure_access_external_chains.md

```diff
@@ -0,0 +1,207 @@
+The {{open_autonomy}} framework supports that agent services access different external chains
+by defining certain configuration parameters appropriately.
+
+For example, if you are developing a service to be run in Ethereum, you might first want to use
+a development environment like [Hardhat](https://hardhat.org/) or [Ganache](https://trufflesuite.com/ganache/) as your testbed, then migrate to a public testnet like [Ropsten](https://ropsten.etherscan.io/) or [Görli](https://goerli.net/), and ultimately run your service in the Ethereum main chain.
```

Comment:
We should no longer list Ropsten, it is being deprecated.

Reply:
fixed